plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityDisplay.java

```diff
@@ -0,0 +1,60 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.entity.ItemDisplay;
+import org.bukkit.entity.TextDisplay;
+
+public class EntityDisplay extends EntityProperty<ElementTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name display
+    // @input ElementTag
+    // @description
+    // For an item display entity this is the model transform it will display, can be any of <@link url https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/ItemDisplay.ItemDisplayTransform.html>.
+    // For a text display entity this is its text alignment, can be any of <@link url https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/TextDisplay.html>.
+    // -->

```

Comment:
should use `@synonyms EntityTag.display_transform, EntityTag.text_alignment`

Reply:
Added synonyms for both